assets/docs/CallmeMehdi/CallmeMehdi.md

```diff
@@ -0,0 +1,13 @@
+# Publisher CallmeMehdi
```

Comment:
Please use all lowercase for the publisher name. Same for the directory and filename.